xs2a-impl/src/main/java/de/adorsys/aspsp/xs2a/domain/CashAccountType.java

```diff
@@ -20,6 +20,9 @@
 
 @ApiModel(description = "CashAccountType", value = "Cash Account Type")
 public enum CashAccountType {
-    // todo documentation doesn't have any definition. hhttps://git.adorsys.de/adorsys/xs2a/aspsp-xs2a/issues/45
```

Comment:
Yes, we already have a definition for this enum and we will implement it in the next version of our project. Thank you